miniapps/contact/tribol.cmake

```diff
@@ -0,0 +1,42 @@
+#------------------------------------------------------------------------------
+# CMake executable path: /opt/homebrew/bin/cmake
```

Comment:
The `config-build.py` of `tribol` takes this path instead of using my environment.

We can probably do
`# CMake executable path: cmake`
it worked for me

Reply:
good point. Fixed